stdlib/public/Backtracing/Backtrace.swift

```diff
@@ -16,14 +16,22 @@
 
 import Swift
 
-@_implementationOnly import OS.Libc
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+internal import Darwin
```

Comment:
Darwin is pretty heavy, is it possible to import one of the modules under it like `errno_h` or `stdio_h` if they're available?

Reply:
Maybe. That might involve importing a number of modules, mind, and it'll only really affect the compile time of the backtracing library (since we aren't exporting anything from them). So it's worth experimenting with, for sure, but we can probably defer that for now.

Reply:
I was thinking more like pulling in dependencies you don't want rather than compile time. i.e. do you really want bsm and nurses and all the other bs that Darwin gives you? Or do you just need a few cstdlib things?